cartographer/mapping_3d/local_trajectory_builder.h

```diff
@@ -40,10 +40,24 @@ namespace mapping_3d {
 // without loop closure.
 class LocalTrajectoryBuilder {
  public:
+  // The following two structs are used for representing the result of adding
+  // range data in local SLAM (using 'LocalTrajectoryBuilder::AddRangeData()').
+  // 'nullptr' is returned for 'MatchingResult' in case if the range data
+  // accumulation is not yet complete. Otherwise, a pointer is returned to a
+  // 'MatchingResult' instance which contains the accumulated 'RangeData',
+  // the estimated pose, time and an 'InsertionResult' pointer, which is
+  // 'nullptr' if the motion filter has blocked the accumulated range data from
```

Comment:
Similar here: Put this next to the `insertion_result` member of `MatchingResult`, e.g.
```
    // 'nullptr' if dropped by the motion filter.
    std::unique_ptr<const InsertionResult> insertion_result;
```